docs/dev/new_check_howto.md

```diff
@@ -440,19 +470,33 @@ For Agent versions >= 6.12:
 "C:\Program Files\Datadog\Datadog Agent\bin\agent.exe" integration install -w /path/to/wheel.whl
 ```
 
-[1]: https://virtualenv.pypa.io/en/stable
-[2]: https://github.com/DataDog/integrations-core/blob/master/docs/dev/python.md
-[3]: https://github.com/DataDog/integrations-extras
-[4]: https://github.com/DataDog/integrations-core/tree/master/datadog_checks_dev
-[5]: https://docs.datadoghq.com/developers/metrics/agent_metrics_submission/
-[6]: https://github.com/DataDog/datadog-agent/blob/6.2.x/docs/dev/checks/python/check_api.md
-[7]: https://docs.pytest.org/en/latest
-[8]: https://tox.readthedocs.io/en/latest
-[9]: https://datadoghq.dev/integrations-core/setup/#ddev
-[10]: https://docs.datadoghq.com/developers/integrations/check_references/#configuration-file
-[11]: https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file
-[12]: https://docs.datadoghq.com/developers/integrations/check_references/#metrics-metadata-file
-[13]: https://docs.datadoghq.com/developers/integrations/check_references/#service-check-file
-[14]: https://packaging.python.org/en/latest/tutorials/packaging-projects/
-[15]: https://docs.datadoghq.com/agent/
-[16]: https://datadoghq.dev/integrations-core/meta/config-specs/
+## Integration publishing checklist
+
+After you complete your integration, refer back to this list to make sure you've got everything you need.
+
+To consider an Agent-based integration complete, and thus ready to be included in the core repository and bundled with the Agent package, several prerequisites must be met:
+
+- A `README.md` file with the correct format and contents
+- A battery of tests verifying metrics collection
+- A `metadata.csv` file listing all of the collected metrics
+- A complete `manifest.json` file
+- If the integration collects Service Checks, the `service_checks.json` must be complete as well
```

Comment:
Should we mention that the CI will also perform some validations and it should be green?